lessons/python/scikit-learn/UofTCoders_ML.ipynb

```diff
@@ -102,19 +99,17 @@
     "## Iris dataset\n",
     "### Samples of 3 different species of Iris flower with 4 features (sepal and petal lengths and widths)\n",
     "### We will learn:\n",
-    "- Displaying feature relationships, correlations\n",
-    "- Dimensionality reduction using PCA\n",
-    "- Scaling the data\n",
-    "- Classification and clustering algorithms\n",
-    "- Compare to another image dataset of handwritten digits"
+    "    -Displaying feature relationships, correlations\n",
+    "    -Dimensionality reduction using PCA\n",
+    "    -Scaling the data\n",
+    "    -Classification and clustering algorithms\n",
+    "    -Compare to another image dataset of handwritten digits"
```

Comment:
These bullets will render correctly if you remove the indent before the `-` and add one space after the `-`.